### ISSUE TITLE
Tutorial content does not match project files

### ISSUE THREAD

**Issue by paralaks** [spring-guides/gs-service-registration-and-discovery#28]
Tutorial has deviated from the project in a few places:
* Bootstrap file is mentioned in tutorial but missing in project files
* Client dependencies incorrectly mention adding eureka-server which does not exist in project's pom.xml
* Although I got the apps working, I have no idea what is going on. The rest endpoint does not respond to requests.

**Comment by SuYehTarn** [comment]
I have encountered the same problem.
Thank you for mentioning the difference between pom.xml.
I found the dependencies for the client are not Eureka Server and Eureka Discovery Client, which are mentioned in the tutorial document.
The correct dependencies are Spring Web and Eureka Discovery Client, which are listed in the pom.xml file of the client in this repo.
I specify these dependencies with [Spring Initializr](https://start.spring.io) and it works.
